pkg/golinters/goanalysis/runners.go

```diff
@@ -118,6 +126,13 @@ func buildIssues(diags []Diagnostic, linterNameBuilder func(diag *Diagnostic) st
 	return issues
 }
 
+func formatSuggestedEdit(suggestedEdit string) string {
+	if suggestedEdit == "" {
+		return ""
+	}
+	return fmt.Sprintf("\nsuggestion\n```\n%s\n```\n", suggestedEdit)
```

Comment:
hmm, the `suggestion` keyword must be inside the code block: ` ```suggestion ` otherwise it will not work on GitHub, but I think this is not good idea at all, since we have different output formats.
so it should be implemented as I did in the PR #1778 

you can take a code from that PR and adjust it for different output formats

Reply:
Sure @SVilgelm .

Reply:
@SVilgelm I went through your PR. It does look like a better approach. I'll write the code suggestion printing method for all printers. It will take some time. I'm hoping I can get it done in about two days.